src/libraries/System.Text.Json/tests/Serialization/TestClasses.SimpleTestClass.cs

```diff
@@ -122,7 +122,7 @@ public class SimpleTestClass : ITestClass
                 @"""MyDateTime"" : ""2019-01-30T12:01:02.0000000Z""," +
                 @"""MyDateTimeOffset"" : ""2019-01-30T12:01:02.0000000+01:00""," +
                 @"""MyGuid"" : ""1B33498A-7B7D-4DDA-9C13-F6AA4AB449A6""," +
-                @"""MyUri"" : ""https://github.com/dotnet/corefx""," +
```

Comment:
These changes are causing failures in the System.Text.Json tests

Reply:
```
Assert.Equal() Failure\r\nExpected: https://github.com/dotnet/corefx\r\nActual: https://github.com/dotnet/runtime
```

Reply:
Thanks Jan,  I'll look into this.